src/components/GlobalMasthead.vue

```diff
@@ -1,29 +1,94 @@
 <script setup lang="ts">
-import { useI18n } from 'vue-i18n';
+import { computed } from 'vue';
 
-import { useViewBreakpoints } from '@/hooks/viewBreakpoints';
+import { useCurrency } from '@/hooks/currency';
+import { useIsAuthenticated } from '@/hooks/isAuthenticated';
 
 import UserInfo from '@/components/UserInfo.vue';
 
-const { t } = useI18n();
-const { isMobile } = useViewBreakpoints();
+const { balance } = useIsAuthenticated();
+const { formatter: formatAsFiat } = useCurrency();
+
+const { value } = balance;
+
+const formattedBalance = computed(() => {
+  return value ? formatAsFiat.value(value) : '';
+});
```

Comment:
```suggestion
const formattedBalance = computed(() => {
  return balance.value ? formatAsFiat.value(balance.value) : '';
});
```

This is an intended way of accessing the ref internal value